### ISSUE TITLE
Incorrect rank calibration.

### ISSUE THREAD

**Issue by Zen1th69** [ValveSoftware/Dota2-Gameplay#8873]
### Description

I'm so sad, I cried today because I was Ancient 1 and I work 12 hours a shift. when I played my 24 calibration matches(12 won 12 lost) I got Archon 4, can you believe archon 4 straight from ancient 1, that's sad on many levels because I lost almost 1000 mmr just because some dumb A.I can't judge who the best medusa is in town.

Steam ID: 76561198052854349

### Example Match ID (and possibly Timestamp)

luma na puss ati

**Comment by ukxavier** [comment]
I was at divine 1 before new patch and then in calibration i picked carry role and won 11 games and lost 5 and still got -100 or -150 mmr from my previous rank. I even had good kda, high tower damage, 700+ gpm xpm still ended up losing mmr how is this possible? need help.

steam id : 76561198081976141

**Comment by Zen1th69** [comment]
> 

Omg man thats so sad, i feel sorry for you and for the opponent's thats gonna face you since your gameplay is of divine player and you're playing in ancient league so technically thats a smurf made by valve dumb A.I. 

Likewise, im ancient and im owning these kiddos of archons but the point is that my teammates are archon aswell hence, i have to 1v9 all of the time.

**Comment by SDSG33** [comment]
Yep it's batshit stupid. I won 11 lost 5 for the recalibration and it placed me -80 mmr than my previous one while my friend who was in the same party for those 16 games skipped 5 ranks. What's the point of winning when you'll just lose mmr.

**Comment by cyrusforoughi** [comment]
Sorry to hear that. I had a similar problem. See post #8686. Went 17-2 and lost 600MMR (Immortal down to Divine 3). I hope there is some fix/adjustment.
